homeassistant/components/config/entity_registry.py

```diff
@@ -238,6 +238,7 @@ def _entry_dict(entry: er.RegistryEntry) -> dict[str, Any]:
         "hidden_by": entry.hidden_by,
         "icon": entry.icon,
         "id": entry.id,
+        "unique_id": entry.unique_id,
```

Comment:
Why do we want unique ID? This is only used in combination with platform and component to generate an ID

Reply:
It's used by the script api : `config/script/config/${id}`.